src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/DataStreamTest/DataStreamTest.cs

```diff
@@ -1566,7 +1572,7 @@ private static void ReadStream(string connectionString)
                         }
 
                         // Once Reader is closed
-                        action = (() => stream.Read(buffer, 0, buffer.Length));
+                        action = (() => stream.ReadExactly(buffer, 0, buffer.Length));
```

Comment:
Been having issues with this in my test branch:
https://dev.azure.com/sqlclientdrivers/public/_build/results?buildId=102097&view=logs&j=12f36a2b-3298-5283-ba0d-570258d2371e&t=5c7d99dc-33ce-5a1c-b601-d5438d2e70c8&l=217